test/jit/call.txt

```diff
@@ -0,0 +1,153 @@
+;;; TOOL: run-interp-jit
+(module
```

Comment:
We're missing tests involving a JITted function calling into an interpreted function. It might be a bit tough to fit that into the existing test framework (since we need a way to ensure a function isn't JITted even though `--trap-on-failed-comp` is enabled), but I think such tests would be invaluable, since that situation seems to have quite a few gotchas.

Reply:
Agreed.

I haven't thought of a clean way of implementing JIT-to-interpreter call tests yet, given the test framework we have. Currently, there is no way of blacklisting functions so that they don't get JIT compiled. Since the compilation threshold is 1 for all JIT tests, the called function is necessarily going to be JIT compiled on its first call. Increasing the threshold might be a start but the test would have to be carefully crafted so that the caller is called multiple times (so it can get JIT compiled) and without having it call the callee function (so that it won't get JIT compiled).